src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {ThemeProvider} from '../ThemeProvider';
+import {Card} from '../Card';
+import {TextContainer} from '../TextContainer';
+import {Thumbnail} from '../Thumbnail';
+import {Stack} from '../Stack';
+import {Button} from '../Button';
+import {TextStyle} from '../TextStyle';
+
+export interface OrderDetailsProps {}
+
+export class OrderDetails extends React.PureComponent<OrderDetailsProps> {
+  render() {
+    return (
+      <ThemeProvider
+        theme={{
+          colorScheme: 'dark',
+          colors: {
+            surface: '#020c1d',
+            secondary: '#103560',
+            interactive: '#009973',
+          },
+        }}
+      >
+        <Card title="Order details">
```

Comment:
A card optionally takes footer actions like `primaryFooterAction`. In your case, I think it may make sense to use the stack because you also have left-aligned text. I am curious if it's possible to mix and match actions with the sections.